project/Build.scala

```diff
@@ -915,34 +917,39 @@ object Build {
       unmanagedSourceDirectories in Test +=
         baseDirectory.value.getParentFile.getParentFile / "shared/src/test/scala",
 
-      if (isGeneratingForIDE) {
-        unmanagedSourceDirectories in Test +=
-          baseDirectory.value.getParentFile.getParentFile / "shared/src/test/scala-ide-stubs"
```

Comment:
@sjrd, any hints on how to deal with IDEs here? Do we need to make `sourceGenerators` dependent on `isGeneratingForIDE`? Or is this even something we'd want to upstream? (e.g. ` buildInfoStub := pathToFile`).

Reply:
I'm not sure how this is going to interact with the IDE stubs. IIUC, with the changes proposed here, generating for IDEs will *both* included the stubs *and* the `BuildInfo`-generated files. This is the worst of both worlds:
* we need to compile the std lib when importing in the IDE, and
* we have compile errors in the IDE because there are duplicate definitions.

I don't really know what we can do about it. If we can't convince `BuildInfo` not to generate its files when generating for IDEs, we should at least get rid of the IDE stubs altogether.

Reply:
> If we can't convince BuildInfo not to generate its files when generating for IDEs, we should at least get rid of the IDE stubs altogether.

I have added a commit (to be squashed) that does this. LMK what you think.

Just TBC, I do not use isGeneratingForIDE support (IIUC you do), so I can't really judge whether this "is worth it".